watchdog/schemas/server.yaml

```diff
@@ -0,0 +1,22 @@
+%YAML 1.1
+---
+"$schema": "http://json-schema.org/draft-06/schema#"
+title: Watchdog daemon role
+description: |
+  Watchdog daemon, server role.
+type: object
+additionalProperties: false
+
+required:
+- enabled
+
+properties:
+  enabled:
+    description: |
+      Enables Watchdog daemon service.
+    type: boolean
+  timeout:
```

Comment:
You did not describe kernel parameters that you support

Reply:
Fixed...